### PR TITLE
Perform actions project details page

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
https://www.notion.so/saeloun/Project-details-page-the-user-should-be-able-to-perform-actions-a1127e3af5c3468d986fc9e60b1c1a0a
## Summary

The user can perform actions on project details page.

## Preview

https://www.loom.com/share/7e508b689ed344208fff3de20b369d3d
## Type of change
- New feature

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Comment by supriya3105** [comment]
@onkar-saeloun  Please review this PR.  
@apoorv1316 This should be tested and merged today.

File: app/javascript/src/components/Projects/Details/index.tsx [hunk 1]
```diff
@@ -5,41 +5,70 @@ import projectAPI from "apis/projects";
...
-      .then(resp => {
+    await projectAPI
+      .show(params.projectId)
+      .then((resp) => {
         setProject(unmapper(resp.data.project_details));
-      }).catch(() => {
-      // Add error handling
+      })
+      .catch(() => {
+        // Add error handling
```
Comment (apoorv1316) [inline app/javascript/src/components/Projects/Details/index.tsx:71]: @Shruti-Apte Can you wrap the async function in try-catch? Also why we are leaving the catch block empty?

**Review by apoorv1316 — APPROVED** [review]
LGTM from functional perspective.

**Review by onkar-saeloun — APPROVED** [review]

**Comment by onkar-saeloun** [comment]
LGTM after resolving Apoorv's comment

**Comment by supriya3105** [comment]
@Shruti-Apte Please resolve the comments on this PR first ?

**Pull request merged**